### PR TITLE
No1 Update Home Page

### PR DESCRIPTION
**Pull request by arranfw**
<html>
<body>
<google-sheets-html-origin>

Change # | Summary | Description
-- | -- | --
1 | Update text on landing page | The landing page has added more text and needs to be updated

</body>
</html>

image

### CODE REVIEW

File: components/ServiceBCLink.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Link from 'next/link';
+
+const serviceBCNumber = '250-111-1111';
```
Comment (arranfw) [inline components/ServiceBCLink.tsx:3]: Single place to update this number throughout the app

File: pages/index.tsx [hunk 2]
```diff
@@ -17,24 +18,69 @@ export default function Home() {
...
-        candidate, you will be directed to call a nurse who will complete a more detailed
-        assessment.
-      </p>
-      <InfoBox content='If for any reason you are unable to fill out the questionnaire online and would like tocomplete it over the phone with support, please call ServicesBC at xxxxxxxx and someone can assist you.' />
+
+      <div className='flex flex-col gap-4 mb-4'>
+        <p>
+          Treatment is available for some people who have tested positive for COVID-19. These
+          treatments have been approved for use in Canada to prevent worsening symptoms in
+          vulnerable patients. . They have been shown to prevent COVID-19 from progressing in high
```
Comment (fw-noel) [inline pages/index.tsx:26]: Is the double period here a typo on their end?

File: pages/index.tsx [hunk 3]
```diff
@@ -17,24 +18,69 @@ export default function Home() {
...
+          anti-viral treatment.
+        </p>
+
+        <p>
+          Please consider and answer each question carefully. Your responses will help to determine
+          if it would be safe for you to receive an anti-viral treatment for COVID-19. If it looks
+          like you might benefit from the treatment you will be directed to call ServicesBC so
+          thatthey can startthe assessment process. They will take the time to review these
+          questions in more detailwith you, and may support your connection to a healthcare team who
+          can access your medication and health information, seek more information,and make
```
Comment (fw-noel) [inline pages/index.tsx:73]: Another typo on their end? No space on the oxford comma

Comment (fw-noel) [reply pages/index.tsx:73]: And `detailwith`

Comment (fw-noel) [reply pages/index.tsx:73]: And Pam's comment about `ServiceBC` vs. `ServicesBC`

**Review by fw-noel — APPROVED** [review]

**Pull request merged**